### PR TITLE
Patryk Jump Revision

### PR DESCRIPTION
**Pull request by Validissimus**
-Revised filljumpmove to work with cyclic behavior
-Revised hasjumpmove to work with cyclic behavior
-Tested it to see if it works normally/as it should, appears to be functioning properly after testing (more testing may be needed though)

### CODE REVIEW

File: src/Model/Move.java [hunk 1]
```diff
@@ -12,6 +12,24 @@ public Move(int oldinput){
...
 	}
+	public Move(int oldcoord, int newcoord, int kill)
+	{
+		old_coordinate = oldcoord;
+		new_coordinate = newcoord;
+		_kills = new LinkedList<Integer>();
+		this.addToKills(kill);
+	}
+	
+	public Move(int oldinput, int newcoord, List<Integer> moves){
```
Comment (TaurusSilver) [inline src/Model/Move.java]: I noticed that you made a new constructor here but it does not appear to be used anywhere. Please remove if it is unnecessary or reorder it to fit with the other constructors if it is needed.

File: src/Test/BoardTest.java [hunk 2]
```diff
@@ -7,6 +7,17 @@
 public class BoardTest {
 	
 	@Test
+	public void double_jump(){
```
Comment (TaurusSilver) [inline src/Test/BoardTest.java]: This isn't a proper test case. Please write a tests for common scenarios and edge cases. You should have two test cases: One indicating that the cyclic bug is fixed and one indicating that normal double jump functions are correct.

File: src/Model/Move.java (hunk 1 above)
Comment (Validissimus) [inline src/Model/Move.java]: It's used in hasJumpMoves

Comment (TaurusSilver) [inline src/Model/Move.java]: Gotcha
